src/evict/evict_stat.c

```diff
@@ -117,11 +117,11 @@ __evict_stat_walk(WT_SESSION_IMPL *session)
 }
 
 /*
- * __wt_evict_stat_walk --
+ * __wt_evict_cache_stat_walk --
```

Comment:
Question: Why did we add the word `cache` into this function name? We just split it out from the cache logic

Reply:
This is not the stat function that we split from cache_conn.c into evict_conn.c; that one is still neatly separated in `__wt_evict_stats_update`.

What we have here is the dhandle-level stat walk function, which updates both evict and cache stats together (EvictCacheWalkStats). I’ve changed the function name to better reflect this. We can consider separating it out in the future, but this logic hasn't seen much use in the past 10 years, so I thought to leave it as is (as it's much more complicated to seperate out neatly).

Reply:
SGTM 👍. Resolving